websites/F/FormulaOneTv/presence.ts

```diff
@@ -0,0 +1,79 @@
+const presence = new Presence({ clientId: "916438450952097834" }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    browsing: "general.browsing"
+  });
+
+function getEpochInMs(): number {
+  return Math.floor(+new Date() / 1000);
+}
+
+function parseTimeToMilliseconds(length: string): number {
+  const [seconds, minutes, hours] = length
+    .split(":")
+    .reverse()
+    .map((val) => parseInt(val, 10));
+
+  return seconds + (minutes || 0) * 60 + (hours || 0) * 3600;
+}
+
+const presenceData: PresenceData = {
+  largeImageKey: "logo_512"
+};
+
+async function setWatchingVideoActivity() {
+  const titleText = document.querySelector(".media-body h5")?.textContent;
+  presenceData.details = titleText
+    ? `Watching ${titleText}`
+    : "Watching a race";
+
+  // Video is playing
+  const playButton = document.querySelector(".bmpui-ui-playbacktogglebutton");
+  if (playButton?.ariaPressed === "true") {
```

Comment:
```suggestion
  if (playButton?.ariaPressed) {
```

Reply:
`ariaPressed` prop is not a boolean  but a string with either `"true"` or `"false"` values. This code would not produce the desired result.

![image](https://user-images.githubusercontent.com/39736248/144941218-6ddd2ceb-909f-4529-a344-5a409c8d1b58.png)